base/uk.ac.stfc.isis.ibex.scriptgenerator/src/uk/ac/stfc/isis/ibex/scriptgenerator/generation/ParametersConverter.java

```diff
@@ -0,0 +1,100 @@
+package uk.ac.stfc.isis.ibex.scriptgenerator.generation;
+
+import java.util.List;
+import java.util.Map;
+import com.google.gson.Gson;
+
+/**
+ * Holds the content for converting user data to json format and also used for
+ * holding content from json file
+ * 
+ * @author mjq34833
+ *
+ */
+public class ParametersConverter {
+
+	String version;
+	String scriptGeneratorVersion;
+	String date;
```

Comment:
date and time of save or load?

Reply:
date and time of save. I am not sure what are we going to use date and time of load for? Do we need it?